bootstrap/config/rbac/role.yaml

```diff
@@ -109,24 +68,18 @@ rules:
 - apiGroups:
   - cluster.x-k8s.io
   resources:
-  - machines
-  - machines/status
+  - machinesets
+  - machinesets/status
```

Comment:
Note for myself to add this to RBAC PR

Reply:
Also we don't seem to look for machine deployments anymore, so could remove that